libs/ui/package.json

```diff
@@ -10,18 +10,18 @@
     "story:preview": "histoire preview"
   },
   "devDependencies": {
-    "@histoire/plugin-vue2": "^0.16.1",
-    "@vitejs/plugin-vue2": "^2.2.0",
-    "@oruga-ui/oruga": "^0.6.0",
+    "@histoire/plugin-vue": "^0.16.1",
+    "@vitejs/plugin-vue": "^4.2.3",
+    "@oruga-ui/oruga-next": "^0.6.0",
     "eslint": "^8.45.0",
     "eslint-plugin-vue": "^8.7.1",
     "histoire": "^0.16.2",
-    "vite": "^3.2.7",
-    "vue": "2.7.14"
+    "vite": "^4.2.0"
   },
   "dependencies": {
-    "@google/model-viewer": "^1.12.1",
     "bulma": "0.9.4",
-    "vue-audio-visual": "2.5.0"
+    "vue": "^3.2.47",
+    "vue-audio-visual": "2.5.0",
+    "@google/model-viewer": "^1.12.1"
```

Comment:
Why is this strangely sorted?

Reply:
because adding dependencies manually hehe